app/containers/MassEnergizeSuperAdmin/Teams/CreateNewTeamForm.js

```diff
@@ -43,11 +43,14 @@ const styles = (theme) => ({
  }) {
   const [parents, setParents] = React.useState([]);
 
+  const isSuperAdmin = auth?.is_super_admin;
```

Comment:
what happens if isSuperAdmin is null?  or undefined?

Reply:
@BradHN1 your code still breaks. You did not import "auth" as a prop before using it inside "CreateNewTeamForm".